baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -84,6 +84,13 @@ export class AssetServiceClient {
     }
     opts.servicePath = opts.servicePath || servicePath;
     opts.port = opts.port || port;
+
+    // users can override the config from client side, like retry codes name, bundling options.
+    // The detailed structure of the clientConfig can be found here: https://github.com/googleapis/gax-nodejs/blob/master/src/gax.ts#L546
+    // The way to override client config for Showcase API:
+    //
+    // const customConfig = {"interfaces": {"google.showcase.v1beta1.Echo": {"methods": {"Echo": {"retry_codes_name": "idempotent", "retry_params_name": "default", "bundling": {..}}}}}}
```

Comment:
the final nit, let's completely remove `bundling` from the example, it will only confuse users :)

Reply:
cool, thanks!